script/1_0_2current_en_script/Common/00_00.txt

```diff
@@ -1,133 +1,133 @@
+This feeling, it was as though fire was coming out of my eyes.
+As soon as I placed my hands on the firm bed, a searing pain cut deep into me.
 <Page6>
 "--- ---"
-I take a deep breath and inspect my body while I'm lying down.
+I took a deep breath and inspected my body while I was lying down.
 <Page7>
-My hands are covered in bandages.
-My left leg is also wrapped in a white plaster.
-It's awfully tight around my forehead, so it's probably wrapped in bandages too.
-There's a large, thick adhesive bandage on my chest, covering up anything inconvenient.
+My hands were covered in bandages.
+My left leg was also wrapped in a white plaster.
+It's been awfully tight around my forehead, so it was probably wrapped in bandages too.
+There was a large, thick adhesive bandage on my chest, covering up anything inconvenient.
 <Page8>
-There is a single tube inserted in my left arm.
-I could vaguely understand that this is an IV drip.
+There was a single tube inserted in my left arm.
+I could vaguely understand that this was an IV drip.
 <Page9>
-I guess it was because I was looking around half-asleep before waking up.
+I guessed it was because I was looking around half-asleep before waking up.
 <Page10>
 The darker room that smelled of medicine,
 the numerous tubes extending from the bandage on my chest,
 the scary man in the suit that came to visit,
 the sight of my sister being scolded by that man,
 and even the sight of the doctor frolicking like a child,
-all of which, I remember as though it were a dream.
+all of which, I remembered as though it were a dream.
 <Page11>
 "Huh? You're not actually awake, are you!?"
-I hear an unfamiliar girl's voice.
-I don't even have the strength to answer her.
+I heard an unfamiliar girl's voice.
+I didn't even have the strength to answer her.
 <Page12>
-Panicking, the woman darts out of the room calling for a doctor.
+Panicking, the woman darted out of the room calling for a doctor.
 Pitter patter pitter patter.
-I get that she's in a rush, but she really shouldn't run in a hospital.
+I got that she was in a rush, but she really shouldn't be running in a hospital.
 <Page13>
 "------"
 I couldn't raise my body, so I turned my head to look up at the window.
-My eyeballs throb as though they are about to splinter in two.
-My brain feels like it's been scrambled with a knife.
+My eyeballs throbbed as though they were about to split in two.
+My brain felt like it's been scrambled with a knife.
 <Page14>
 Truly... an unbelievable blue sky.
 More than the white rays of sunlight, it was the harshness of that unblemished blue that pained my eyes.
 <Page15>
 A few days later.
 For a while after I woke up, I couldn't move from the bed.
 <Page16>
-Looking up, I can see the lofty skies outside same as ever.
-The curtains flutter and sway.
-A dry breeze strokes my cheek.
-The stainless, immaculate blue makes my eyes spin.
+Looking up, I could see the lofty skies outside being the same as ever.
```

Comment:
Slightly uncomfortable wording. Perhaps "Looking up, I could see the sky outside, lofty as ever" or "I could see the lofty skies outside, just as always" / "outside, just as I always could"